atlasdb-tests-shared/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionTest.java

```diff
@@ -92,10 +92,11 @@ protected boolean supportsReverse() {
         return true;
     }
 
-    protected static final int GET_RANGES_THREAD_POOL_SIZE = 16;
+    public static final int GET_RANGES_THREAD_POOL_SIZE = 16;
+    public static final int DEFAULT_GET_RANGES_CONCURRENCY = 4;
```

Comment:
On second (third?) thought, I think we can remove these and just reuse the constants from `TransactionTestConstants` - it should be visible for tests in `atlasdb-tests-shared`.